manim/mobject/mobject.py

```diff
@@ -1309,16 +1309,18 @@ def __init__(self, *mobjects, **kwargs):
 
 
 class _AnimationBuilder:
-    def __init__(self, mobject):
+    def __init__(self, mobject, generate_target=True):
         self.mobject = mobject
+        if generate_target:
+            self.mobject.generate_target()
 
     def __getattr__(self, method_name):
         from ..animation.transform import _MethodAnimation
```

Comment:
This shouldn't be necessary anymore, correct? Or perhaps it would be better to have a separate method within `_AnimationBuilder` for creating the animation that could be called in `Scene.compile_animations`? That would better organize the code I think, though this local import would still be a thing

Reply:
This would probably also create a cleaner interface for custom animations, +1 for a dedicated method creating the animation.